.github/workflows/content-share-integration.yml

```diff
@@ -17,12 +17,23 @@
     integ-content-share:
       name: Content Share Integration Test
       runs-on: ubuntu-latest
+      if: ${{ github.head_ref != 'release' }}
 
       steps:
         - name: Checkout Package
           uses: actions/checkout@v2
           with:
             fetch-depth: 0
+        - name: Get Current Package version
```

Comment:
Since, this is repeating now for all steps can we add a github action or a script to generate this tarball prior and re-use everywhere?

Reply:
It might be easier to understand what is happening if we leave it the way it is - if we put it in a script and then run the script from the github action, it might not be as easy to see the order of steps from the github action - you'd have to go to the script and then see what the script is doing.

I'm going to replace the Get Current Package Version steps with our own script - I think its best practice to not use third party github actions if we don't need to. I'll put this one in a script since its straight forward what it does.